--- origsrc/caja-extensions-1.8.0/configure.ac	2014-03-01 06:13:11.000000000 -0600
+++ src/caja-extensions-1.8.0/configure.ac	2014-03-05 12:54:52.580343700 -0600
@@ -99,7 +99,6 @@ if test "x$enable_sendto" = "xyes"; then
         GTHREAD_REQUIRED=2.6.0
         DBUS_REQUIRED=1.0
         DBUS_GLIB_REQUIRED=0.60
-        GUPNP_REQUIRED=0.13
         PKG_CHECK_MODULES(SENDTO,
                 glib-2.0    >= $GLIB_REQUIRED
                 gio-2.0     >= $GLIB_REQUIRED
@@ -108,8 +107,7 @@ if test "x$enable_sendto" = "xyes"; then
                 gobject-2.0 >= $GLIB_REQUIRED
                 gtk+-$GTK_API_VERSION >= $GTK_REQUIRED
                 dbus-1      >= $DBUS_REQUIRED
-                dbus-glib-1 >= $DBUS_GLIB_REQUIRED
-                gupnp-1.0   >= $GUPNP_REQUIRED)
+                dbus-glib-1 >= $DBUS_GLIB_REQUIRED)
         AC_SUBST(SENDTO_CFLAGS)
         AC_SUBST(SENDTO_LIBS)
         AC_DEFINE_UNQUOTED(GAJIM_SHARE_DIR, "$prefix/share/gajim", [path to gajim share dir])
--- origsrc/caja-extensions-1.8.0/sendto/plugins/Makefile.am	2013-10-17 15:16:44.000000000 -0500
+++ src/caja-extensions-1.8.0/sendto/plugins/Makefile.am	2014-03-05 12:52:09.506016400 -0600
@@ -1,10 +1,8 @@
 SUBDIRS = . \
-	caja-burn \
 	emailclient \
 	gajim \
 	pidgin \
-	removable-devices \
-	upnp
+	removable-devices
 
 NST_COMMON_SOURCES = nst-common.c nst-common.h
 
